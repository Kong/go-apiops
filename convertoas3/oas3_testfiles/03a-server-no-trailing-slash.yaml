# servers[0].url doesn't have a trailing slash
# if service.url is only a hostname, with no path nor
# trailing /

openapi: 3.0.3

info:
  title: Server with no trailing slash
  version: v1
servers:
  - url: "https://example.com"

paths:
  /path:
    get:
      operationId: myId
      responses:
        "200":
          description: OK
