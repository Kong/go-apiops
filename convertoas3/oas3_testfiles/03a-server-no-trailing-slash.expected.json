{
  "_format_version": "3.0",
  "services": [
    {
      "host": "example.com",
      "id": "5434f850-a000-5ad5-8772-1a3cd10811e6",
      "name": "server-with-no-trailing-slash",
      "path": "/",
      "plugins": [],
      "port": 443,
      "protocol": "https",
      "routes": [
        {
          "id": "d44c95fe-dc1c-596c-8b4a-e18ff50607c4",
          "methods": [
            "GET"
          ],
          "name": "server-with-no-trailing-slash_myid",
          "paths": [
            "~/path$"
          ],
          "plugins": [],
          "regex_priority": 200,
          "strip_path": false,
          "tags": [
            "OAS3_import",
            "OAS3file_03a-server-no-trailing-slash.yaml"
          ]
        }
      ],
      "tags": [
        "OAS3_import",
        "OAS3file_03a-server-no-trailing-slash.yaml"
      ]
    }
  ],
  "upstreams": []
}